Exercise_3/Coord.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exercise_3
+{
+    public class Coord
```

Comment:
Need to refactor
- Move classes `Coord` `Sys` `Weathers` to namespace `Model`
- Move classes `ICityWeather`  to namespace `Service`
- Rename `Weathers` to `WeatherInfo`  (not plural form)

Reply:
Done